.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+tab_width = 4
+
+[*.gradle]
```

Comment:
Do we even have such a file? :0

Reply:
Whoops. This is my goto editorconfig for java and I have been only using gradle so far.. I will delete this.